io.github.nest-desktop.yml

```diff
@@ -0,0 +1,88 @@
+id: io.github.nest-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+  - org.freedesktop.Sdk.Extension.typescript
+command: nest-desktop-launch
+separate-locales: false
+rename-desktop-file: nest-desktop.desktop
+rename-icon: nest-desktop
+rename-appdata: nest-desktop
+finish-args:
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to save files locally
+  - --filesystem=home
+  # XShm access
+  - --share=ipc
+  # Needs to talk to the network:
+  - --share=network
+  # Display access (X11 or Wayland)
+  # - --socket=fallback-x11
+  - --socket=system-bus
+  - --socket=x11
+  # - --socket=wayland
+modules:
+  - name: nest-desktop
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/nest-desktop/nest-desktop
+        tag: v4.0.0
+        commit: 81bab14049ed82abe05b7ca38e57d7d82e582964
+      - type: file
+        path: nest-desktop-launch
+      - type: file
+        path: nest-desktop.appdata.xml
+      - type: file
+        path: nest-desktop.desktop
+      - type: file
+        path: icons/nest-desktop-256x256.png
+      - type: file
+        path: icons/nest-desktop-512x512.png
```

Comment:
these 3 should be in https://github.com/nest-desktop/nest-desktop

Reply:
the last 3?

Reply:
yes

![image](https://github.com/user-attachments/assets/9a510dd6-b504-4268-9404-4a4a48358985)

Reply:
I moved all files (icons, metainfo and desktop) to upstream repo (`https://github.com/nest-desktop/nest-desktop`)